docs/src/api/params.md

```diff
@@ -213,6 +213,11 @@ Whether to ignore HTTPS errors during navigation. Defaults to `false`.
 
 Toggles bypassing page's Content-Security-Policy.
 
+## context-option-baseUrl
+- `baseUrl` <[string]>
+
+Prefixes the URL on navigations when a path (no URL) is passed to [`method: Page.goto`].
```

Comment:
Please add an example here with relative and absolute page.goto(). Also, strictly speaking, path+query+hash also works, so perhaps say "no origin" instead of "path"?